designs/virtualization/4.5/Scheduling/Scheduling-section/Scheduling-section.md

```diff
@@ -0,0 +1,17 @@
+---
+parent: Virtualization
+version: 4.5
+---
+
+# Scheduling Section
+
+Creating a new section in the VM details page, Overview tab
+('Scheduling': If, where and how this workload will be placed within the cluster's nodes)
+
+![Operator details page](img/VM-page-0-0.jpg)
+
+Introducing a new section within the Details tab: `Scheduling and Resources Requirements`.
+This section comes right after the Virtual Machine Details section.
+This section contains all the properties that are set from this workload and affect this workload's scheduling.
+The two columns are sorted so the left column is dedicated for label-based kubernetes scheduling policies.
```

Comment:
I wonder if because you are ordering them like this we should have sub headings for each column, Would that help or just add noise?

Reply:
Note sure the name fits. If we don't have a good solution it might be clearer without it.
WDYT?
<img width="860" alt="Screen Shot 2020-03-26 at 10 39 00" src="https://user-images.githubusercontent.com/48942365/77626473-16ddfa80-6f4e-11ea-9274-e94e45603c55.png">

Reply:
I think this is a nice addition. Especially b/c we could see users who wouldn't know the difference. Maybe the spacing is a bit too much between the heading and the content of the column?

Reply:
Committed, WDYT?

Reply:
did you change the space? it looks the same to me 🤷‍♂